### PR TITLE
Sc 33186 add video sections back

### PR DESCRIPTION
**Pull request by Anakhom**
https://app.shortcut.com/xanaduai/story/33186/update-meta-info-for-hubs-and-add-video-section

- Updated meta images for all hubs 
- Added video sections back with thumbnalis 

Demos:
- QML
Screen Shot 2023-02-03 at 1 08 26 PM
- QC 
Screen Shot 2023-02-03 at 1 44 24 PM

- QChem
Screen Shot 2023-02-03 at 1 08 58 PM

### CODE REVIEW

File: quantum-machine-learning.rst [hunk 1]
```diff
@@ -82,16 +82,16 @@ Quantum machine learning
             </div>
         </section>
 
-        <!--
         <hr class="pb-4 text-dark">
+
         <section>
             <div class="d-flex flex-wrap justify-content-center">
-                <h2 class="fw-normal text-center mt-1 mb-5 border-0">Sit back and explore quantum computing with our curated selection of expert videos.</h2>
+                <h2 class="fw-normal text-center mt-1 mb-5 border-0">Sit back and explore quantum machine learning with our curated selection of expert videos.</h2>
```
Comment (josh146) [inline quantum-machine-learning.rst:89]: 👍

**Review by josh146 — APPROVED** [review]
Brilliant, thanks @Anakhom! I went through the rendered site, tried all links etc, everything looks great.

**Review by ashishks0522 — APPROVED** [review]
Code looks good! Thanks @Anakhom 🚀

**Review by LucasSilbernagel — APPROVED** [review]
Looks good to me!

**Comment by ikurecic** [comment]
Awesome, thank you! Although, am I the only one who finds the 'Watch' link a little confusing? The link is obviously already there on the thumbnail.
@BM7878 Would it make some sense to instead of 'Watch' have a 'Watch more' link that leads to the YT playlist of PL videos?

**Comment by Anakhom** [comment]
> Awesome, thank you! Although, am I the only one who finds the 'Watch' link a little confusing? The link is obviously already there on the thumbnail. @BM7878 Would it make some sense to instead of 'Watch' have a 'Watch more' link that leads to the YT playlist of PL videos?

@ikurecic I might have some input here. So this entire video card in itself is a link that is dedicated to have one main link. It might be  wonky (in terms of the implementation) and a bit not user friendly if we point "watch more" somewhere else other than the video from the card due to the entire card being one link, not just the thumbnail.  I would advocate for leaving the current card as is.

**Review by BM7878 — APPROVED** [review]

**Pull request merged**